.env_example

```
@@ -99,4 +99,4 @@ YES_EMOJI=""
 # OPTIONAL - What should be your bot's emoji for every failed sentence?
 # Example: NO_EMOJI="❌"
 # Default: ❌
-NO_EMOJI=""
\ No newline at end of file
+NO_EMOJI=""
```

```suggestion
NO_EMOJI=""
```
Do not remove this empty/blank last line